locales-lt-LT.xml

```diff
@@ -29,22 +27,22 @@
     <term name="anonymous">anonimas</term>
     <term name="anonymous" form="short">anon.</term>
     <term name="at"></term>
-    <term name="available at">available at</term>
+    <term name="available at">adresas</term>
     <term name="by"></term>
-    <term name="circa">circa</term>
-    <term name="circa" form="short">ca.</term>
-    <term name="cited">cituojama pagal</term>
+    <term name="circa">apie</term>
+    <term name="circa" form="short">apie</term>
+    <term name="cited">žiūrėta</term>
     <term name="edition">
       <single>leidimas</single>
       <multiple>leidimai</multiple>
     </term>
     <term name="edition" form="short">leid.</term>
-    <term name="et-al">et al.</term>
+    <term name="et-al">ir kt.</term>
     <term name="forthcoming">ruošiamas</term>
     <term name="from"></term>
-    <term name="ibid">ibid.</term>
+    <term name="ibid">ten pat</term>
```

Comment:
Both `ibid` (Latin form) and `ten pat` (Lithuanian) are allowed and can be found.